src/utils/sound_loader.py

```diff
@@ -1,7 +1,21 @@
 from src.conf import config
-
+import math
+import numpy as np
+import scipy.signal as sig
 
 class SoundLoader:
     def __init__(self):
         # tak się dobiera do ustalonych parametrów (wszystkie są w configurator.py)
```

Comment:
Ten print jest niepotrzebny, on w sumie tylko miał zademonstrować jak korzystać z configa.
Najlepiej gdyby to wyglądało tak:
def __init__(self):
    pass